lib/modules/manager/nuget/artifacts.ts

```diff
@@ -55,16 +59,17 @@ async function runDotnetRestore(
     packageFileName,
   );
 
+  const dotnetVersion =
+    config.constraints?.dotnet ??
+    (await findGlobalJson(packageFileName))?.sdk?.version;
```

Comment:
The schema only enforces that it's a string. Should we worry about there being some value like `"foo"` or `"lts"` or even just `"6"` which isn't a valid constraint?

Reply:
I think the docs only mention a full version, so I think everything else is an error and should not happen 🤷‍♂️